### DISCUSSION TITLE
Uninstall and re-install, or update

### DISCUSSION THREAD

**Discussion by Doug140** [noisetorch/NoiseTorch#322] [Q&A (How can I ...?)] [answered]
I have the "Possible Compromise, Remove Immediately" message on NoiseTorch still. I click "Update" and get an "Update failed" message. Should I uninstall NoiseTorch and then re-install it per instructions in the readme, or is there a way to update directly?

**Comment by AXDOOMER** [comment, marked as answer]
Please download the latest update from here: https://github.com/noisetorch/NoiseTorch/releases/tag/v0.12.0

**Reply by Doug140** [reply]
I uninstalled the old version per instructions here:
https://github.com/noisetorch/NoiseTorch/blob/master/README.md

rm ~/.local/bin/noisetorch
rm ~/.local/share/applications/noisetorch.desktop
rm ~/.local/share/icons/hicolor/256x256/apps/noisetorch.png 

Downloaded the file and installed, per instructions here:
https://github.com/noisetorch/NoiseTorch/blob/master/README.md

Everything works great! Thank you all for your work and your help.
